File: benchmark/corpus/03_div_zero/f_remainder_in_loop.py
# A remainder inside a loop over a list; the step comes from input.
import sys


def bucket_all(xs, step):
    hits = 0
    for x in xs:
        if x % step == 0:  # DANGER
            hits += 1
    return hits


line = sys.stdin.readline().strip()
try:
    step = int(line)
except ValueError:
    print("not a number: " + line)
else:
    print("%d hits" % bucket_all([4, 8, 15, 16], step))

File: benchmark/corpus/03_div_zero/f_remainder_in_loop.js
// A remainder inside a loop over a list; the step comes from input.
function bucketAll(xs, step) {
  let hits = 0;
  for (const x of xs) {
    if (x % step === 0) hits++; // DANGER
  }
  return hits;
}

const line = (await new Response(Deno.stdin.readable).text()).trim();
const step = Number(line);
if (Number.isNaN(step)) console.log("not a number: " + line);
else console.log(bucketAll([4, 8, 15, 16], step) + " hits");
